ryft-ui/src/main/java/com/ryftpay/android/ui/extension/StringExtension.kt

```diff
@@ -1,27 +1,38 @@
 package com.ryftpay.android.ui.extension
 
+private val TITLES = arrayOf("MISS", "MR", "MRS", "MS")
```

Comment:
I noticed some of the billing address names can be the 'name on card' which can contain titles too... this won't cover everything but I figured it's reasonable for now